src/pages/journal/components/indicators/indicators.scss

```diff
@@ -4,6 +4,14 @@ indicators {
     img {
       width: 29px;
       height: 25px;
+      .text-zoom-large & {
```

Comment:
Is the idea that we configure these CSS classes individually for each of our UI components?

Reply:
The typography should scale fine without changes, but there might be times when you want more control (like making images match the text sizes) and this approach would work. Possibly not ideal. Ideally we would use rem/em to scale from a base font, but the base font is always fixed in this approach.

Reply:
And I used the `.text-zoom-large &` ampersand syntax as it allows the extra classes to stay near the element being overridden. Just seems neater.